packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -278,7 +278,7 @@ export interface AutocompleteProps<
    */
   slotProps?: {
     clearIndicator?: Partial<IconButtonProps>;
-    paper?: PaperProps;
+    paper?: Partial<PaperProps>;
```

Comment:
All the props of Paper component are optional, so we don't need `Partial` utility. Or is there any issue you are facing?
```suggestion
    paper?: PaperProps;
```

Reply:
Okay, alright. Let's keep it consistent.